client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/RestService.java

```diff
@@ -126,6 +128,14 @@ public RestService(String baseUrlConfig) {
   public void setSslSocketFactory(SSLSocketFactory sslSocketFactory) {
     this.sslSocketFactory = sslSocketFactory;
   }
+  
+  public void setHttpConnectTimeoutMS(int timeout) {
```

Comment:
These two methods are currently unused without code changes. As I mention above, I suggest removing them and using static variables as reasonable defaults.